recommendation/templates/recommendation/recommendation_rating.html

```diff
@@ -0,0 +1,189 @@
+                    <th><input type="radio" name="rating_oral" value=2></th>
+                    <th><input type="radio" name="rating_oral" value=3></th>
+                    <th><input type="radio" name="rating_oral" value=4></th>
+                    <th><input type="radio" name="rating_oral" value=5></th>
+                    <th><input type="radio" name="rating_oral" value=6></th>
+                    <th><input type="radio" name="rating_oral" value=7></th>
+                    <th><input type="radio" name="rating_oral" value=0></th>
+                </tr>
+                <tr>
+                    <th> Perserverance toward goals </th>
+                    <th><input type="radio" name="rating_goals" value=1 required></th>
+                    <th><input type="radio" name="rating_goals" value=2></th>
+                    <th><input type="radio" name="rating_goals" value=3></th>
+                    <th><input type="radio" name="rating_goals" value=4></th>
+                    <th><input type="radio" name="rating_goals" value=5></th>
+                    <th><input type="radio" name="rating_goals" value=6></th>
+                    <th><input type="radio" name="rating_goals" value=7></th>
+                    <th><input type="radio" name="rating_goals" value=0></th>
+                </tr>
+                <tr>
+                    <th> Ability to get along with others </th>
+                    <th><input type="radio" name="rating_socialization" value=1 required></th>
+                    <th><input type="radio" name="rating_socialization" value=2></th>
+                    <th><input type="radio" name="rating_socialization" value=3></th>
+                    <th><input type="radio" name="rating_socialization" value=4></th>
+                    <th><input type="radio" name="rating_socialization" value=5></th>
+                    <th><input type="radio" name="rating_socialization" value=6></th>
+                    <th><input type="radio" name="rating_socialization" value=7></th>
+                    <th><input type="radio" name="rating_socialization" value=0></th>
+                </tr>
+                <tr>
+                    <th> Ability to analyze problems and formulate solutions </th>
+                    <th><input type="radio" name="rating_analyzing" value=1 required></th>
+                    <th><input type="radio" name="rating_analyzing" value=2></th>
+                    <th><input type="radio" name="rating_analyzing" value=3></th>
+                    <th><input type="radio" name="rating_analyzing" value=4></th>
+                    <th><input type="radio" name="rating_analyzing" value=5></th>
+                    <th><input type="radio" name="rating_analyzing" value=6></th>
+                    <th><input type="radio" name="rating_analyzing" value=7></th>
+                    <th><input type="radio" name="rating_analyzing" value=0></th>
+                </tr>
+            </table>
```

Comment:
Nit - 
The fields have repeating structure with only 1 value differing, using forloop will be a lot less code and easier readability. Again, just something that would be a good to have and not necessary to have.